src/azure-cli/azure/cli/command_modules/apim/_params.py

```diff
@@ -314,6 +314,19 @@ def load_arguments(self, _):
         c.argument('file_path', options_list=['--file-path', '-f'],
                    help='File path specified to export the API.')
 
+    with self.argument_context('apim policy') as c:
+        c.argument('api_id', arg_type=api_id)
+        c.argument('operation_id', help='Operation identifier within an API. Must be unique in the current API Management service instance.')
+        c.argument('specification_path', help='Contents of the Policy as defined by the format.')
+        c.argument('policy_format', help='Format of the policy content. Allowed formats: rawxml, rawxml-link, xml, xml-link')
```

Comment:
Could we use enumeration to define these valid values?

Reply:
of course!